cpp/daal/include/services/internal/sycl/math/mkl_blas.h

```diff
@@ -63,9 +63,10 @@ struct MKLGemm
         const MKL_TRANSPOSE transamkl = transa == math::Transpose::Trans ? MKL_TRANS : MKL_NOTRANS;
         const MKL_TRANSPOSE transbmkl = transb == math::Transpose::Trans ? MKL_TRANS : MKL_NOTRANS;
 
-        cl::sycl::buffer<algorithmFPType, 1> a_sycl_buff = a_buffer.toSycl();
-        cl::sycl::buffer<algorithmFPType, 1> b_sycl_buff = b_buffer.toSycl();
-        cl::sycl::buffer<algorithmFPType, 1> c_sycl_buff = c_buffer.toSycl();
+        cl::sycl::buffer<algorithmFPType, 1> a_sycl_buff = a_buffer.toSycl(status);
```

Comment:
I believe it's better to check after each `toSycl(status)` operation.

Reply:
It increases number of checks for normal flow, but in case of error doesn't add more information to the status. AFAIK, this approach commonly used in the lib